ICSharpCode.Decompiler/Disassembler/DisassemblerHelpers.cs

```diff
@@ -317,10 +317,10 @@ public static void WriteOperand(ITextOutput writer, object operand)
 			
 			VariableReference variableRef = operand as VariableReference;
 			if (variableRef != null) {
-				if (string.IsNullOrEmpty(variableRef.Name))
+				if (string.IsNullOrEmpty(variableRef.ToString()))
```

Comment:
Nit: `variableRef.ToString()` won't be `null` like `variableRef.Name` could have been, so you can just delete this conditional and always take the `else` branch.